guides/common/modules/proc_enabling-the-satellite-tools-repository.adoc

```diff
@@ -10,7 +10,8 @@ Continue with a procedure below according to the operating system version for wh
 
 == [[enabling-repos-rhel9-rhel8]]{RHEL} 9 & {RHEL} 8
 
-To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Enabling_the_Client_Repository_rhel_9_{context}[] or xref:CLI_Enabling_the_Client_Repository_rhel_8_{context}[].
+To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Enabling_the_Client_Repository_rhel_9_{context}[Enabling the Client Repository on RHEL 9] or 
+xref:CLI_Enabling_the_Client_Repository_rhel_8_{context}[Enabling the Client Repository on RHEL 8].
```

Comment:
```suggestion
To use the CLI instead of the {ProjectWebUI}, see the procedure relevant for your {RHEL} version:
* xref:CLI_Enabling_the_Client_Repository_rhel_9_{context}[]
* xref:CLI_Enabling_the_Client_Repository_rhel_8_{context}[]
```